src/pdf/pdf.ts

```diff
@@ -549,16 +549,11 @@ export class PDF_ extends ExtendedPDF {
 
   private _renderQRCode(): void {
 
-    const qrcode = generateQRCode(this._data, utils.mm2pt(46));
-
+    const qrcode = generateQRCode(this._data, "pdf", utils.mm2pt(67), this._marginTop + utils.mm2pt(17), utils.mm2pt(46));
 
     //-- Add QR Code
 
-    this.addPath(qrcode, utils.mm2pt(67), this._marginTop + utils.mm2pt(17))
-      .undash()
-      .fillColor("black")
-      .fill();
-
+    this.fillColor("black").addContent(qrcode).fill();
```

Comment:
Can you flip around `this.fillColor("black")` and `addContent(qrcode)`. It makes more sense that way or is at least more in line with the rest of the code.
```suggestion
    this.addContent(qrcode).fillColor("black").fill();
```